lib/types.ts

```diff
@@ -13,17 +13,22 @@ export enum ApiVersion {
   Unstable = 'unstable',
 }
 
+export const LIBRARY_NAME = 'Shopify API Library';
```

Comment:
For the `X-SDK-Variant` header, we'd prefer a value that's lower case and hyphenated instead of using spaces. Since this `LIBRARY_NAME` is used in the user agent value, maybe we can do something like this while setting the variant header or create a new const:

```
[ShopifyHeader.StorefrontSDKVariant]: LIBRARY_NAME.toLowerCase().split(' ').join('-'),
```

Reply:
I can do that!